packages/auth/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Auth",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "dependencies": {
+    "@auth0/auth0-spa-js": "^1.22.1",
+    "mobx": "^6.6.1",
+    "mobx-react-lite": "^3.4.0",
+    "qs": "^6.11.0",
+    "react": "^17.0.0"
```

Comment:
I think this should be a `peerDependency` because React blows up if we somehow wind up with more than one version in the runtime

Reply:
Should this package receive the latest? "react": "^18.2.0"

Reply:
hmm probably not since I don't think we are actually using that anywhere? (unless you are in JC) but you can specify multiple ranges I think if you join them with `||` if you want to accept 17.x and 18.x

Reply:
Just took a quick look and we're still 17.x - I'll keep it at ^17.